app/graphql/cache.ts

```diff
@@ -29,5 +30,33 @@ export const cache = new InMemoryCache({
         },
       },
     },
+    ConsumerAccount: {
```

Comment:
When I attempted this last time I wrote a script to generate the possible types from the schema on the server https://github.com/GaloyMoney/galoy-mobile/compare/main...refactor-transactions-screen#diff-d3e97a63b7fd33227d3577ef3a18ec8793bf860730677e9f7f3382be13d0cc8f.

I was then able to pass in the generated file to the possibleTypes config - https://github.com/GaloyMoney/galoy-mobile/compare/main...refactor-transactions-screen#diff-05ff1f30a72721bd2ff7481a4e51b12592d8d7ed89c5ac459c25c9c9d2505314